.buildkite/scripts/integration_tests.sh

```diff
@@ -5,10 +5,20 @@ WORKSPACE="$(pwd)"
 TMP_FOLDER_TEMPLATE_BASE="tmp.elastic-package"
 
 cleanup() {
+    local error_code=$?
+
+    if [ $error_code != 0 ] ; then
+        if [ -f ${GOOGLE_APPLICATION_CREDENTIALS+x} ]; then
+             google_cloud_logout_active_account
+        fi
+    fi
```

Comment:
Ensure that in case of failure, the active account is logged out.

Reply:
perfect! we check the variable and if it exists we check the file!